src/themes/styles.js

```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from "styled-components";
+
+import * as color_theme from "./colors";
+
+export const themed_link = (theme) => {
+  // takes theme color, returns link styling
+  return { textDecoration: "none", color: theme.special };
+};
+
+export const MapcGlobalStyle = createGlobalStyle`
+  body {
+    font-family: "Calibre-Regular, SwiftNeueLTPro-Regular";
```

Comment:
We should set these up differently. In general, I think we'll want to use sans-serif fonts for body text and serif for headers/titles etc., and we should [fall back to system defaults](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family) if for some reason our fonts can't be loaded. So, something more like:
```css
p {
  font-family: "Calibre-Regular", sans-serif;
}

h1,h2,h3,h4,h5,h6 {
  font-family: "SwiftNeueLTPro-Regular", serif;
}
```

We could also add in some of the other advanced font css features mentioned in that CSS-Tricks article to enable kerning:
```css
body {
  -moz-font-feature-settings:    "kern" 1; 
  -ms-font-feature-settings:     "kern" 1; 
  -o-font-feature-settings:      "kern" 1; 
  -webkit-font-feature-settings: "kern" 1; 
  font-feature-settings:         "kern" 1;
  font-kerning: normal;
  font-family: "Calibre-Regular", sans-serif;
}
```